packages/eslint-plugin/src/rules/jsx-pragma/index.ts

```diff
@@ -27,13 +27,58 @@ const findReactDeclarationWithDefaultImport = (
   return undefined;
 };
 
+function createFixer(context: Rule.RuleContext, source: SourceCode, options: Options) {
+  return function* fix(fixer: Rule.RuleFixer) {
```

Comment:
This was just lifted and shifted, no changes outside of moving `const pragma = ...` inside here.

Reply:
ah cool, thank you for refactoring it